bases/br_me_rais/microdados_vinculos/table_config.yaml

```
@@ -0,0 +1,492 @@
+table_id: microdados_vinculos # AUTO GENERATED
+dataset_id: br_me_rais # AUTO GENERATED
+
+version: v1.0
+
+last_updated: 2020-12-24 # AUTO GENERATED
+
+# Descreva a tabela. Essas são as primeiras frases que um usuário vai ver.
+# Você não precisa ser muito conciso. Sinta-se a vontade para dar exemplos de
+# como usar os dados.
+# Se souber, liste também aplicações: pesquisa, apps, etc. que usem os dados.
+description: |  # REQUIRED
+    Microdados públicos dos vínculos de emprego na RAIS.
```

@rdahis trocaria aqui para:
```suggestion
    Microdados públicos dos vínculos de emprego na RAIS. Base desidentificada, isto é, que não inclui identificadores únicos de linha. Cada linha representa um vínculo - por isso indicamos este como nível de observação mesmo que não conste como coluna.
```